docs/verify/simulators.mdx

```diff
@@ -6,15 +6,15 @@ description: Overview of simulators
 
 IBM Quantum features a collection of high-performance simulators for prototyping quantum circuits and algorithms.
 
-To view available simulators, on the upper right corner of the screen, click the Application switcher ( ![switcher](/images/migration/switcher-small2.png) ), select **Compute resources** to view the [Compute resources page,](https://quantum-computing.ibm.com/services/resources?services=simulators) then click the Simulators tab.
+To view available simulators, on the upper right corner of the screen, click the Application switcher ( ![switcher](/images/migration/switcher-small2.png) ), select **Compute resources** to view the [Compute resources page,](https://quantum-computing.ibm.com/services/resources?services=simulators) then click All simulators.
 
-The default choice of simulator should be the `simulator_statevector` as it is a general-purpose solution method. The following simulation methods are currently available, and support a maximum of 300 circuits and 8192 shots per job.
+The following simulation methods support a maximum of 300 circuits and 8192 shots per job. Find more information on each simulator below, including its type, a description, the number of qubits it simulates, whether it includes noise modeling, a list of supported gates, and a minimum code example. The `simulator_statevector` is a good default choice since it is a general-purpose solution method. 
```

Comment:
```suggestion
The following simulation methods support a maximum of 300 circuits and 8192 shots per job. Find more information on each simulator below, including its type, a description, the number of qubits it simulates, whether it includes noise modeling, a list of supported gates, and how to call them using Qiskit Runtime. The `simulator_statevector` is a good default choice since it is a general-purpose solution method. 
```